ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -685,6 +685,14 @@ function upgrade() {
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
+
+    conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]?.release.version' | xargs)
```

Comment:
Just like what we discussed in slack, I think we can do more:
1. introduce a dedicated step to do the check, opt it into `openshift-upgrade-qe-sanity`, so that we can exit the whole job in advance.
2. sounds like `cucushift-chainupgrade-toimage` also need this check.